.github/workflows/crowdin-sync-translations.yml

```diff
@@ -1,29 +1,29 @@
-name: Weekly Sync Translations job
+# Retrieves non-English translations from Crowdin and creates a PR with new changes
+
+name: Sync Crowdin Translations
 
 on:
-  workflow_dispatch:
```

Comment:
Why the possibility of custom execution has been removed?

Reply:
My mistake, I saw the colon in the end and thought that this is an empty placeholder configuration. I've added it back.